Documentation/InputSimulation/InputSimulationService.md

```diff
@@ -173,4 +174,13 @@ For manipulating objects with two hands at the same time the persistent hand mod
 4. Click and hold _left mouse button_ to interact
 5. Rotate the camera again to manipulate the object
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/6841rRMdqWw" class="center" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen />
\ No newline at end of file
+<iframe width="560" height="315" src="https://www.youtube.com/embed/6841rRMdqWw" class="center" frameborder="0" allow="accelerometer; encrypted-media; gyroscope; picture-in-picture" allowfullscreen />
+
+## Eye tracking
+
+Eye tracking simulation can be enabled by checking the __Simulate Eye Position__ option in the
+[Input Simulation Profile](#enabling-the-input-simulation-service). This should not be used with GGV
+style interactions (so ensure that __Hand Simulation Mode__ is set to _Articulated_).
+
+[This document](../EyeTracking/EyeTracking_BasicSetup.md#simulating-eye-tracking-in-the-unity-editor)
```

Comment:
i'd remove this sentence and put the link to the eye tracking simulation in the first sentence ( see suggested change)